src/components/Loading/docs/LoadingBox.md

```diff
@@ -1,4 +1,3 @@
-### LoadingBox example:
 Light:
```

Comment:
was removing the above line intentional?

Reply:
Yup, it was repetitive since styleguide adds a heading already